### DISCUSSION TITLE
Display quiz timing (Moodle 4.1 and SEB)

### DISCUSSION THREAD

**Discussion by AntonT76** [SafeExamBrowser/seb-win-refactoring#612] [General]
Hello,
I have just noticed that the opening and closing times of the quiz are not displayed when using the SEB (3.5.0) in Moodle 4.1.3 (Boost Union Theme) as a student if a student try to open a quiz which will start in future. It only shows the number of attempts, time limit, etc.

Is there any possibility to display this times?

thx, br Anton

**Comment by dbuechel** [comment]
I guess that's more a Moodle than a SEB related question, so I'd recommend asking it in a Moodle forum.
